cache/FIFO_FH.h

```diff
@@ -303,7 +303,7 @@ FIFO_FHCache<TKey, TValue, THash>::FIFO_FHCache(size_t maxSize, double chunkRati
 template <class TKey, class TValue, class THash>
 bool FIFO_FHCache<TKey, TValue, THash>::construct_ratio(double FC_ratio) {
   // Check valid ratio
-  assert(FC_ratio <= 1 && FC_ratio >=0);
+  assert(FC_ratio <= 1 && FC_ratio > 0);
```

Comment:
My reasoning is, if FC_ratio is 0 it shouldn't be constructing an FC cache to begin with and would also replace need to check `(FC_size != 0)`